deepmedic/logging/accuracyMonitor.py

```diff
@@ -146,12 +148,91 @@ def reportAccuracyForLastSubepoch(self) :
             numOfFpInSubep = numOfRnInSubep - numOfTnInSubep
             
             logStrClass = logStr + ", " + classString + ":"
-            self.log.print3(logStrClass+"\t mean accuracy:   \t"+ strFl4fNA(meanAccClassOfSubep, self.NA_PATTERN)+"\t=> (TruePos+TrueNeg)/All-Predicted-Voxels = "+str(numOfTpInSubep+numOfTnInSubep)+"/"+str(numOfRpInSubep+numOfRnInSubep))        
-            self.log.print3(logStrClass+"\t mean sensitivity:\t"+ strFl4fNA(meanAccOnPosOfSubep, self.NA_PATTERN)+"\t=> TruePos/RealPos = "+str(numOfTpInSubep)+"/"+str(numOfRpInSubep))
-            self.log.print3(logStrClass+"\t mean precision:\t"+ strFl4fNA(meanPrecOfSubep, self.NA_PATTERN)+"\t=> TruePos/(TruePos+FalsePos) = "+str(numOfTpInSubep)+"/"+str(numOfTpInSubep+numOfFpInSubep))
-            self.log.print3(logStrClass+"\t mean specificity:\t"+ strFl4fNA(meanAccOnNegOfSubep, self.NA_PATTERN)+"\t=> TrueNeg/RealNeg = "+str(numOfTnInSubep)+"/"+str(numOfRnInSubep))
-            self.log.print3(logStrClass+"\t mean Dice:       \t"+ strFl4fNA(meanDiceOfSubep, self.NA_PATTERN))
-            
+            self.log.print3(logStrClass+"\t mean accuracy:   \t" +
+                            strFl4fNA(meanAccClassOfSubep, self.NA_PATTERN) +
+                            "\t=> (TruePos+TrueNeg)/All-Predicted-Voxels = " +
+                            str(numOfTpInSubep+numOfTnInSubep) + "/" +
+                            str(numOfRpInSubep+numOfRnInSubep))
+            self.log.print3(logStrClass+"\t mean sensitivity:\t" + strFl4fNA(meanAccOnPosOfSubep, self.NA_PATTERN)+"\t=> TruePos/RealPos = "+str(numOfTpInSubep)+"/"+str(numOfRpInSubep))
+            self.log.print3(logStrClass+"\t mean precision:\t" + strFl4fNA(meanPrecOfSubep, self.NA_PATTERN)+"\t=> TruePos/(TruePos+FalsePos) = "+str(numOfTpInSubep)+"/"+str(numOfTpInSubep+numOfFpInSubep))
+            self.log.print3(logStrClass+"\t mean specificity:\t" + strFl4fNA(meanAccOnNegOfSubep, self.NA_PATTERN)+"\t=> TrueNeg/RealNeg = "+str(numOfTnInSubep)+"/"+str(numOfRnInSubep))
+            self.log.print3(logStrClass+"\t mean Dice:       \t" + strFl4fNA(meanDiceOfSubep, self.NA_PATTERN))
+
+    def log_to_tensorboard(self, metrics_dict, class_string, step_num):
```

Comment:
log_to_tensorboard_ys_for_x(...)

Reply:
I think this is unnecessary. A tensorboard log is always associated with a step, and a log to tensorboard is always of metrics